test/iidm/converter/xml/extensions/LoadMockExtXmlSerializer.cpp

```diff
@@ -29,6 +29,7 @@ LoadMockExtXmlSerializer::LoadMockExtXmlSerializer() :
         VersionsCompatibilityBuilder()
             .put(IidmXmlVersion::V1_0(), {"1.0"})
             .put(IidmXmlVersion::V1_1(), {"1.1", "1.2"})
+            .put(IidmXmlVersion::V1_2(), {"1.2"})
```

Comment:
Don't know how it works in java without adding this version in the map, unless it throws an exception when calling `LoadMockExtXmlSerializer::getVersion()`...

Reply:
It's not allowed to have `1.2` compatible with XIIDM v1.1 and v1.2
In Java, this class is used only to check the compatibility support. The test runs only in v1.1. Maybe you should adapt the test.